lib/builder-webpack5/src/presets/module-federation.ts

```diff
@@ -0,0 +1,113 @@
+import type { Configuration, EntryObject, WebpackPluginFunction } from 'webpack';
+import { logger } from '@storybook/node-logger';
+import WebpackVirtualModules from 'webpack-virtual-modules';
+
+interface VirtualModules {
+  [key: string]: string;
+}
```

Comment:
optional - I guess this could be a `Record` as well but no biggie.

Reply:
Good idea, I'll update it.

Reply:
Done